config.py

```diff
@@ -266,7 +269,7 @@ class configuration:
   ###########################################################################
 
   # A list of the flags for SWIG
-  swig_flags = ['-c++', '-python', '-keyword'] #, '-keyword', '-py3']
+  swig_flags = ['-c++', '-python', '-keyword', '-py3']
 
```

Comment:
The addition of the `-py3` flag seems to be causing the compile error. Perhaps the `py3` flag could be added only if python 3.0 is being used?